src/components/sections/chaoshub/WhyChaoshub.tsx

```diff
@@ -66,7 +71,12 @@ const UsefulLinksContent: React.FC = () => {
             target="_blank"
```

Comment:
This anchor tag can be given a style...or rather you can give the style in index.css to make all anchor tags to not have an underline.